src/components/userProfile/UserProfile.js

```diff
@@ -201,6 +221,14 @@ class UserProfile extends Component {
     );
   }
 
+  handleScroll(event) {
+    if (event.nativeEvent.contentOffset.y < 0) {
+      this.setState({
+        scale: event.nativeEvent.contentOffset.y / -200 + 1,
```

Comment:
Unexpected mix of '-' and '*'

@MechanicKim Suggested applying this new rule
https://github.com/sv-bootcamp/bridgeme-client/pull/143#discussion-diff-93888737R54